src/clients/api/mutations/bridgeXvs/index.ts

```diff
@@ -0,0 +1,44 @@
+import BigNumber from 'bignumber.js';
+import { ContractTransaction, ethers } from 'ethers';
+
+import { NULL_ADDRESS } from 'constants/address';
+import { layerZeroChainIds } from 'constants/layerZero';
+import { XVSProxyOFTDest, XVSProxyOFTSrc } from 'packages/contracts';
+import { ChainId } from 'types';
+
+export interface BridgeXvsInput {
+  tokenBridgeContract: XVSProxyOFTSrc | XVSProxyOFTDest;
+  accountAddress: string;
+  destinationChain: ChainId;
+  amountMantissa: BigNumber;
+  gasValueMantissa: BigNumber;
```

Comment:
I feel this name could be confusing: I first thought this corresponded to the gas fee of the transaction made on the chain of origin (which we let users select in their wallet directly).
Perhaps something like `bridgeFeeMantissa` would be clearer. What do you think?

Reply:
Changed it to `nativeCurrencyFeeMantissa`,  to indicate that the fee will be in BNB, ETH, and so on